docs/user/ppl/functions/datetime.rst

```diff
@@ -711,6 +738,30 @@ Example::
     +---------------------------------+
 
 
+DAY_OF_WEEK
+---------
```

Comment:
```suggestion
DAY_OF_WEEK
-----------
```

Reply:
fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/281/commits/7b927df652fd09ef0e74315eb6a2abfbffe1f8f3